app/assets/stylesheets/_board.scss

```diff
@@ -103,6 +103,7 @@
 
 .column h3 {
   line-height: 20px;
+  min-height: 42px;
```

Comment:
This feels wrong, but without it the header loses its bottom padding if the column is full....unsure why.
<img width="226" alt="chrome_2016-04-08_10-43-52" src="https://cloud.githubusercontent.com/assets/133987/14391384/507c2d40-fd81-11e5-9e7c-e46e2987f494.png">


/cc @JBertling